argonaut/src/main/scala/org/http4s/argonaut/ArgonautInstances.scala

```diff
@@ -32,6 +32,15 @@ trait ArgonautInstances {
   def jsonEncoderWithPrinterOf[A](prettyParams: PrettyParams)
     (implicit encoder: EncodeJson[A]): EntityEncoder[A] =
     jsonEncoderWithPrettyParams(prettyParams).contramap[A](encoder.encode)
+
+  implicit val urlCodec: CodecJson[Uri] = CodecJson(
```

Comment:
This is necessary for knobs' Configured?  I have knobs on all my classpaths but haven't done much with it yet.  (Hangs head in shame.)